src/dpl/include/dpl/Opendp.h

```diff
@@ -212,6 +212,7 @@ class Opendp
   int getRowSiteCount() const { return row_site_count_; }
 
  private:
+  friend class Dpl_IsPlaced_Test;
```

Comment:
Are all tests going to have to be friends?  Shouldn't we test a class through its public interface?

Reply:
In general it's better to test along the public interface, but the public interface for a lot of tools is too broad. I found that testing the isPlaced function was simple enough to demonstrate the new pattern and infrastructure, but is it self not a perfect example of a good test since it's arguably way too simple.